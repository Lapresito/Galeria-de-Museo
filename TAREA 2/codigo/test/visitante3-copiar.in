crearVisitante 31 Casimira Gomez 29
copiarVisitante
liberarVisitante
Fin
