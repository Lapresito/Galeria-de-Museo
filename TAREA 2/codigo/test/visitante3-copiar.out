Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Visitante creadox de forma exitosa.
2> Visitante copiado. Datos de la copia:
Visitante Casimira Gomez
Id: 31
Edad: 29
3> Visitante liberado con exito.
4> Fin.
